boards/arm/mimxrt1170_evk/mimxrt1170_evk_cm7.dts

```diff
@@ -29,6 +29,7 @@
 		zephyr,flash = &is25wp128;
 		zephyr,code-partition = &slot0_partition;
 		zephyr,display = &lcdif;
+		zephyr,keyboard-scan = &touch_controller;
```

Comment:
@CristXu you need to update the table at the bottom of doc/build/dts/api.rst if you're going to add a new chosen node

@galak don't forget to mention this too please

Reply:
@mbolivar-nordic its in that table already - https://docs.zephyrproject.org/latest/build/dts/api/api.html#id2